bootloader/wscript

```diff
@@ -661,6 +661,9 @@ def build(ctx):
     if ctx.env.CC_NAME == 'msvc':
         # Do not strip bootloaders when using MSVC.
         features = ''
+        
+        ctx.env['WINRC_TGT_F'] = '/fo'
+        ctx.env['WINRC_SRC_F'] = ''
```

Comment:
Setting these flags is already done by waf, but only if WINRC is not set, see bottol of waflib/tools/winrc.py. So the correct solution is to not search for WINDRES if CC is msvc in line 399.